### ISSUE TITLE
draklive-install - installation in italian (Bugzilla Bug 503)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#503]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 503
Date: 2014-02-12 18:18:33 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;
To:   Maik Wagner &lt;<none@none.com>&gt;
CC:   @tapwag, siriustheking@yahoo.com, @tpgxyz

Last updated: 2015-07-09 20:41:32 +0000

## Comment 3196
Date: 2014-02-12 18:18:33 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

During installation I choose italian language but these messages are still in english:

1) Please choose your keyboard layout 

2) Please specify which services should run at startup 

3) Please wait while the boot loader ... 

4) Congratulation for installing OpenMandriva. Please ...

## Comment 3491
Date: 2014-03-02 08:27:11 +0000
From: @tapwag

Dear Giorgio,

many thanks for your bug report. We mainly do our localization at the following address

https://www.transifex.com/projects/p/openmandriva-linux/language/it/

The old strings can be found at the following address

https://transifex.mandriva.com/projects/p/mandriva_desktop/r/2011/l/it/

Unfortunately the installer is not yet part of the platform. I assign this bug to me and will get in touch with our developers on how to integrate Transifex.com with the new installer.

## Comment 3492
Date: 2014-03-02 10:19:40 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Hi Mike
thanks for your answer.
If this can help I can give you italian translation of these few sentences.

I visited also the links but I don't know anything about Transifex and it's not so clear to me how it works.

## Comment 3493
Date: 2014-03-02 12:11:44 +0000
From: @tapwag

Hello Giorgio,

thanks for your answer. Basically one can have an account at this localization platform and submit translations or just do proofreading. 

The way it works is that the texts from the source languge are shown on the left and you have a window where you type in the translation.

Unfortunately AFAIK the installer bit hasn't been uploaded yet for Italian and I also had some problems (syntax errors) when uploading the German translations. There must be way to synchronize Transifex.com and the translations which we already have, that's why I leave the bug at NEEDINFO

## Comment 3494
Date: 2014-03-02 13:35:52 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Hi Maik

When installer bit will be uploaded if you don't have any translator I can give a try. I'm not skilled in translation but installation has simple texts and Transifex seems easy to use.
Of course having old translations available helps but in this case is not essential.
So let me know if you need help and I'll get an account in Transifex.

## Comment 3495
Date: 2014-03-02 13:52:15 +0000
From: @tapwag

Dear Giogio,

I wrote an email to the OpenMandriva Council and outlined the issue which is basically getting the old translations (which are there and where Italy has been very active) to the new platform. 

I do not know how the transfer of localisation strings is going to be resolved and we might need some help from Mandriva SA (the company) 

As far as registering on Transifex.com is concerned I am contributing to quite a lot of projects and every little helps: proof-reading of strings for example. 

Have a good day,

Maik "tapwag" Wagner

## Comment 3500
Date: 2014-03-02 14:56:57 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Hi Maik

(In reply to comment #5)
> Dear Giogio,
> 
> I wrote an email to the OpenMandriva Council and outlined the issue which is
> basically getting the old translations (which are there and where Italy has
> been very active) to the new platform. 
It's very helpful in general, even if OMLx 2013 seems already well translated in italian then you could use 2013 translation as starting point. It seems to me that there are few differences between 2013 and 2014. 
This is valid for installation strings too. In 2013 most of them are already translated and installation, till now, isn't changed in 2014. So we could use 2013 strings and just add translation for few messages still in english.

> As far as registering on Transifex.com is concerned I am contributing to
> quite a lot of projects and every little helps: proof-reading of strings for
> example. 
Ok count on me for installation strings. This will be my small contribution.

## Comment 3501
Date: 2014-03-02 15:05:10 +0000
From: @tapwag

(In reply to comment #6)
> It's very helpful in general, even if OMLx 2013 seems already well
> translated in italian then you could use 2013 translation as starting point.
> It seems to me that there are few differences between 2013 and 2014. 

Well there has been one report also from Italy about the starter menu, have a look here:

https://issues.openmandriva.org/show_bug.cgi?id=521

> This is valid for installation strings too. In 2013 most of them are already
> translated and installation, till now, isn't changed in 2014. So we could
> use 2013 strings and just add translation for few messages still in english.
> 

That's the idea. We use English as the source language and translators translate into their native language as it is the language they know best.

I signed up two Italians on the transifex.com team but there hasn't been any decision on who is taking the lead, who does proof-reading etc. 

Are you a member of an Italian OpenMandriva community such as MIB?

## Comment 3509
Date: 2014-03-02 21:19:13 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

I'm a member of MIB and in its forums I often meet the italians involved in bug #521.

## Comment 4528
Date: 2014-03-26 17:05:42 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

In 2014 64 bit beta1 the number of untranslated strings is slightly increased.

If needed I can translate the strings, just let me know how can I send the italian version.

## Comment 4648
Date: 2014-03-28 22:34:28 +0000
From: @tpgxyz

*** Bug #610 has been marked as a duplicate of this bug. ***

## Comment 4652
Date: 2014-03-28 23:22:02 +0000
From: Gus Ballan &lt;<siriustheking@yahoo.com>&gt;

(In reply to comment #10)
> *** Bug #610 has been marked as a duplicate of this bug. ***

Yes, duplicate.
But it's needed to add the Licence, the timezone selection, and the configuration of time and date, all of them untranslated in Spanish.

As it's said in comment 9, the number of untranslated strings has increased, and in the case of Spanish the increment is not subtle.

These problems about translation occur during live mode and probably during install. Running OMV 2014 in live mode doesn't have too much problems with localization. I don't tested installation yet. Live mode was pretty similar with alpha2 (see my post in https://forums.openmandriva.org/en/discussion/543/omv-2014-alpha2-very-first-impressions)

## Comment 4653
Date: 2014-03-28 23:23:40 +0000
From: Gus Ballan &lt;<siriustheking@yahoo.com>&gt;

Sorry, I forgot something.
In previous Mandrivas (2010, 2011) the Licence appeared in Spanish after selecting Spanish in the first page. If the Licence has no changes, I think that translated licences can be extracted from the previous Mandrivas files.

## Comment 5795
Date: 2014-05-03 07:59:01 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Still valid in OpenMandriva 2014 64 bit.

If this help I can translate.

## Comment 6906
Date: 2014-08-25 11:20:39 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Still valid in OpenMandriva 2014.1 alpha!

OpenMandrivaLx.2014.1-kde4.x86_64.iso Id 5556.

## Comment 7218
Date: 2014-09-23 15:51:37 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Still valid in 2014.1 64 bit id 6237.

I don't understand why a simple bug like this isn't yet solved. If you don't solve it now we have to wait till 2015 (or more!!!)

Please let me know if I can send italian translation of string still in english.

## Comment 7301
Date: 2014-10-06 19:42:47 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Still valid in 2014.1 64 bit.
Raised importance hoping will be solved two release after bug opening.

## Comment 7457
Date: 2014-11-12 11:57:00 +0000
From: @tapwag

Dear Giorgio,

thanks for your messages and sorry for taking me so long to get back to you. As far as I understand it the installation routine for 2015 release is going to rewritten. You can have a look at our project management at 

http://project.openmandriva.org

As far as translations are concerned the correct .po-file needs to be found and can be translated either on Transifex.com or using loKalize which is part of KDE. lokalize is a desktop-application. Once the .po-file with the translated strings is ready please attach it here so developers can pick it up and integrate it into the ABF-build-farm.

## Comment 7469
Date: 2014-11-12 21:07:32 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

(In reply to comment #17)
>.. sorry for taking me so long to get back to you.
Don't worry.

> As far as I understand it the installation routine for 2015 release is going
> to rewritten. 
Very good, I think it was necessary. 

>You can have a look at our project management at 
> 
> http://project.openmandriva.org
> 
> As far as translations are concerned the correct .po-file needs to be found
> and can be translated either on Transifex.com or using loKalize which is
> part of KDE. 
Checked. It seems .po isn't yet available. Right?
Is there a way to be automatically alerted when .po file will be available?

>lokalize is a desktop-application. Once the .po-file with the
> translated strings is ready please attach it here so developers can pick it
> up and integrate it into the ABF-build-farm.
Installed Lokalize. I'll wait for translation.

## Comment 7472
Date: 2014-11-13 08:06:09 +0000
From: @tapwag

Dear Georgio,

I am also stepping forward to have a German version of the new installation routine. I just added a feature request to have the installer in different languages:

https://project.openmandriva.org/work_packages/154

Would you still like to do a translation of the old installer or follow the progress of the new installation routine?

## Comment 7488
Date: 2014-11-14 08:29:14 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

(In reply to comment #19)
>I just added a feature request to have the installer in different
> languages:
> 
> https://project.openmandriva.org/work_packages/154
Very good, this is the right way to proceed. This allow to add a translation in any moment.

 
> Would you still like to do a translation of the old installer or follow the
> progress of the new installation routine?
I can do both. 
But I think it's much better wait for new installer. I hope it will include some modification, for example a pause as I asked loooong time ago (bug #276).

If new installer will not be available I can translate old version that's an easy work, it's already partially translated and it can be a last minute work if necessary.

## Comment 8055
Date: 2015-01-20 18:52:55 +0000
From: @tapwag

Some news on translation: First of all I would like to thank everyone who has been contributing on Transifex.com to translate OpenMandriva. I am hoping to get my ABF-account reactivated and then I might be able to make changes to the source code strings online. 

As far as the installer is concerned, it is likely that it is going to be based on this project: http://calamares.github.io/about/ - there is also a Translation team for this installer on transifex.com and you might want to join this team as well. Lots of strings have already been translated there into different languages.

I did a pre-Alpha installation of OpenMandriva LxQt today and would also like to confirm that the license text doesn't show in my native selected language which is German. I am assuming it would be best to open a new bug for the license text.

## Comment 8072
Date: 2015-01-21 16:22:17 +0000
From: @tapwag

Some further news: I am facing the same problem that the installation routine is not fully localized into German. With TPGs advise in IRC I managed to edit some of the strings on OpenMandriva's build platform which is at http://www.abf.io

Here is what I did: I forked the project "drakx" and edited the strings directly in my own repository: 

https://abf.io/tapwag/drakx/tree/master/perl-install/install/help/po

So far I only did German but can add other languages as well. You can also create repositories directly on the ABF.

The po-files are in perl-install/install/help/it.po (for Italian).

Don't forget to send a so called "Pull request" after you made your changes.

Hope that helps,

Maik "tapwag" Wagner

## Comment 8265
Date: 2015-02-05 21:54:42 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

(In reply to comment #22)
> Here is what I did: I forked the project "drakx" and edited the strings
> directly in my own repository: 
> 
> https://abf.io/tapwag/drakx/tree/master/perl-install/install/help/po
Tried to edit here: impossible. 

> So far I only did German but can add other languages as well. You can also
> create repositories directly on the ABF.
I tried but I don't see how.

 
> The po-files are in perl-install/install/help/it.po (for Italian).
Downloaded it.po file but POedit don't open it. Tried with online POeditor: it.po isn't opened. It seems file has error.
How can I translate it?

## Comment 9714
Date: 2015-07-09 20:41:32 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Almost completely solved in 2014.2.
Se evolution of this bug in bug #1266
